dashboard/app/controllers/transfers_controller.rb

```diff
@@ -0,0 +1,53 @@
+class TransfersController < ApplicationController
+  before_filter :authenticate_user!
+  # check_authorization
+
+  # TODO: Copying Progress
```

Comment:
progress is associated only with a student and not with a section so you don't need to copy anything